### PR TITLE
Migration/edit table entity panel to react

### PR DESCRIPTION
**Pull request by sunilyadav840**
Created EditTableEntityPanel/EditTableEntityPanel.tsx (React component for EditTableEntityPanel panel).
Note: [WIP] Cassandra case

https://user-images.githubusercontent.com/79906609/115023947-0d1e9080-9edd-11eb-9597-4ee3b6890e32.mp4

[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/690?feature.someFeatureFlagYouMightNeed=true)

### CODE REVIEW

**Comment by southpolesteve** [comment]
image

The new version of this does not seem to be right for cassandra. The types should already be filled it and not be editable

**Comment by southpolesteve** [comment]
Tested and looks fine for Tables. Needs more work for Cassandra

File: src/Explorer/Explorer.tsx [hunk 1]
```diff
@@ -2273,6 +2277,32 @@ export default class Explorer {
     );
   }
 
+  public openAddTableEntityPanel(queryTablesTab: QueryTablesTab, tableEntityListViewModel: TableListViewModal): void {
+    false
```
Comment (southpolesteve) [inline src/Explorer/Explorer.tsx:2281]: This should be removed. It will always take the false path

File: src/Explorer/Explorer.tsx [hunk 2]
```diff
@@ -2273,6 +2277,32 @@ export default class Explorer {
...
+      "Edit Table Entity",
+      <EditTableEntityPanel
+        explorer={this}
+        closePanel={this.closeSidePanel}
+        queryTablesTab={queryTablesTab}
+        tableEntityListViewModel={tableEntityListViewModel}
+      />
+    );
+  }
+
```
Comment (southpolesteve) [inline src/Explorer/Explorer.tsx:2305]: Remove the old panels too:

```
  public addTableEntityPane: AddTableEntityPane;
  public editTableEntityPane: EditTableEntityPane;
```

**Review by southpolesteve — CHANGES_REQUESTED** [review]

File: src/Explorer/Explorer.tsx (hunk 2 above)
Comment (sunilyadav840) [reply src/Explorer/Explorer.tsx:2231]: Removed

File: src/Explorer/Explorer.tsx (hunk 1 above)
Comment (sunilyadav840) [reply src/Explorer/Explorer.tsx]: Removed

**Comment by sunilyadav840** [comment]
> image
> 
> The new version of this does not seem to be right for cassandra. The types should already be filled it and not be editable

It was WIP. Now Its completed. Please review now.

**Review by southpolesteve — APPROVED** [review]

**Pull request merged**